### DISCUSSION TITLE
Ikea Tradfri E1810 non si connette più al gateway Z2M

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1620] [HUB: Home Assistant]
Ciao a tutti, 

ho un telecomando Ikea Tradfri modello E8110 (quello tondo a 5 tasti) e due deviatori vimar connessi (modello 14592) che fungono anche da router.
Come coordinator ho la chiavetta della ITEAD Sonoff ZigBee 3.0 USB Dongle Plus aggiornata alla versione 20220219.
Versione di Z2M: 1.25.0
Fino a una settimana fa, utilizzavo il pulsante ikea per comandare da remoto uno dei due deviatori vimar.

Qualche giorno fa il telecomando non funzionava più. Non sono riuscito però a capire se è colpa della batteria (che ho trovato scarica) del teelcomando, o dell'aggiornamento della chiavetta ITEAD che avevo aggiornato alla versione precedente a quella attuale.

Fatto sta che dopo vari tentativi di repairing (inizialmente, la piattaforma di Z2M mi segnalava solamente che il telecomando non era più collegato ma non mi era chiaro se lo avesse completamente perso di vista, compariva solo nella lista dei dispositivi), decido di eliminare il dispositivo da Z2M per riaccoppiarlo. Dopo aver sostituito le batterie e tentato anche il reset del dispositivo, niente da fare. LA procedura:

-reset del telecomando (premo 4 volte il tasto di pairing)
-pressione costante per 10 secondi del tasto di pairing finchè il telecomando non emette luce rossa, e mollo il tasto solo quando la luce si spegne)
-Z2M non lo rileva.

provo anche ad omettere la pressione prolungata e mi limito a resettare il dispositivo, ma nulla da fare, z2m non ne vuole sapere di rilevarlo. 
Mi posiziono vicino a uno dei router per l'accoppiamento (precisamente il deviatore del bagno), perchè non voglio creare molta distanza tra il telecomando e la fonte di segnale. MA per la vicinanza, me lo dovrebbe rilevare anche l'altro deviatore situato in studio.
Già provato a resettare Z2M, sia via Putty sia via piattaforma, stccato e riattacato la chiavetta ITEAD, non ne vuole sapere. Nessun errore ,semplicemente è come se non ci fossero dispositivi in attesa di accoppiamento.

Consigli? Mi viene in mente solo di riprovare N volte quello che ho già provato e pregare.
Grazie.
Grazie.

**Comment by Scialla** [comment]
proverei ad associarlo stando vicino al coordinator e non ai router

**Reply by calcioscacchi** [reply]
io ci provo ma poi lo voglio dissociare: il coordinator per quanto potente è lontano dalla postazione dell'interruttore e consumerebbe la batteria.

**Reply by snakuzzo** [reply]
non ho capito che c'entra la batteria.
I device a batteria restano "svegli" per pochi istanti...poi vanno in deep sleep. E tutto questo, a prescindere dalla lontananza, avviene proprio per preservare la durata della batteria.

**Reply by calcioscacchi** [reply]
presto detto. Cercando una soluzione al mio problema, ho letto che maggiore è la distanza, maggiore è il consumo dell'end device. Magari è una boiata eh :)

**Reply by calcioscacchi** [reply]
SOno riuscito a collegarlo da antenna avvicinandomi al coordinator, però che fastidio... perchè il router non rilevava l'interruttore? Misteri...
